database/mongo/seed.js

```diff
@@ -0,0 +1,65 @@
+
+const tags = ['Good for groups', 'Desserts', 'Appetizers', 'Drinks', 'Kid friendly'];
+
+const circleColors = ['#6c8ae4', '#d86441', '#bb6acd', '#df4e96'];
+
+const jabber = new Jabber(buzzWords, 2);
+
+const createReview = (resId) => {
+  const review = {};
+  review.restaurantId = resId;
+  review.firstName = (Math.random() < 0.7) ? faker.name.firstName() : '';
+  if (review.firstName.length) {
+    review.lastName = (Math.random() < 0.7) ? faker.name.lastName() : '';
+  } else {
+    review.lastName = '';
+  }
+  review.city = faker.address.city();
+  review.numReviews = Math.floor(Math.random() * (25 - 1)) + 1;
+  review.overall = Math.floor(Math.random() * (5 - 1)) + 1;
+  review.food = Math.floor(Math.random() * (5 - 1)) + 1;
+  review.service = Math.floor(Math.random() * (5 - 1)) + 1;
+  review.ambience = Math.floor(Math.random() * (5 - 1)) + 1;
+  review.dineDate = moment(faker.date.between('2019-01-01', '2020-01-24')).format('YYYY-MM-DD');
+  // dineDate = faker.date.between('2019-01-01', '2020-01-24');
+  review.noise = Math.floor(Math.random() * (5 - 1)) + 1;
+  review.recommend = Math.random() < 0.7;
+  review.comments = jabber.createParagraph(Math.floor(Math.random() * (50 - 10)) + 10);
+  review.filterTag = tags[Math.floor(Math.random() * tags.length)];
+  review.vip = Math.random() < 0.3;
+  review.color = circleColors[Math.floor(Math.random() * circleColors.length)];
+  return review;
+};
+
+const createReviews = () => {
+  // number of restaurants (will be 100)
+  const restaurantNum = 100;
+  const reviewsArray = [];
+  for (let i = 1; i <= restaurantNum; i += 1) {
+    // number of reviews per restaurant will be 30-90 , random here
+    const reviewCount = Math.floor(Math.random() * (15)) + 5;
+    for (let j = 0; j < reviewCount; j += 1) {
+      reviewsArray.push(createReview(i));
+    }
+  }
+  return reviewsArray;
+};
+
+async function insertDummyData() {
+  await Review.Review.insertMany(createReviews());
+}
+
+insertDummyData()
+  .then(() => {
+    console.log('Database seeded');
+    process.exit();
+  });
```

Comment:
LGTM! This probably could have been a pull request by itself.

Reply:
This was all John's code that I mutated to a point of no return for my data generation.   I had to copy it back in to I could seed his db again and see what the shape was because I had lost it.  I did send a message in slack but I guess I was too late.  My bad.